source/git-util.js

```diff
@@ -110,16 +112,8 @@ exports.commitLogFromRevision = revision => execa.stdout('git', ['log', '--forma
 
 exports.push = () => execa('git', ['push', '--follow-tags']);
 
-const gitVersion = async () => {
```

Comment:
There is no need to merge the two functions together. They are currently split up in order to keep each one single-purposed (and the other `verify...` methods are written this way as well).